test/lit.cfg.py

```diff
@@ -60,4 +62,15 @@
     'llhd-sim'
 ]
 
+# Search for and enable Verilator
+verilator_ext = os.path.join(config.test_source_root, "../ext/bin", "verilator")
```

Comment:
Should this logic be in cmake instead of lit?  This would allow doing some sanity checks to make sure the verilator version is up to date enough and provide config flags so users can specify a path to verilator manually.

Reply:
I didn't put it in cmake since we're not using it in the compilation process and can do the sanity checks in python. But you make a good point about the path. CMake is the way we configure paths, so it should be there.

Reply:
Yeah, the right way to do this is to get cmake to find it, but cmake is annoying :)

Reply:
Agreed about cmake. Why did they invent their own language???